src/components/Select2.js

```diff
@@ -50,7 +50,12 @@ export default class Select2 extends Component {
     return (
       <select multiple={this.props.multiple}>
         {this.props.data.map((item, k) => {
-          return (<option key={'option-' + k}>{item}</option>);
+          if(typeof item === 'string' || ((!!item && typeof item === 'object') && Object.prototype.toString.call(item) === '[object String]')) {
```

Comment:
Please check the [js code style](https://github.com/rkit/react-select2-wrapper/blob/master/.eslintrc): http://jmp.sh/UUbq1Ub (only errors)